clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -226,7 +234,7 @@ class ClioClient(webClient: ClioWebClient,
         .map(token => OAuth2BearerToken(token.getTokenValue))
         .toEither
         .left
-        .map(ex => ParsingOrAuthError(ex.getMessage))
+        .map(ex => AuthError(ex))
```

Comment:
Nice improvement to get the exception instead of just the message. You could use `_` here as well.